nixos/modules/services/web-apps/cryptpad.nix

```diff
@@ -50,5 +76,23 @@ in
         WorkingDirectory = "%S/cryptpad";
       };
     };
+
+    services.cryptpad.settings = {
```

Comment:
this should be the default value of the option, or at least be declared with `lib.mkDefault`

Reply:
Should be fixed now.

Reply:
I'll test this soon.
Is it really impossible to use the `default` key of mkOption ? This way the default value is shown on https://nixos.org/nixos/options.html.